site/_apps/ember-hn.txt

```diff
@@ -0,0 +1,25 @@
+---
+weight: 10
+title: 'HackerNew.io'
+github-title: 'ivanvanderbyl/ember-hackernews-pwa'
+libraries:
+    - name: 'Ember.js'
+module-bundling: 'Broccoli'
+service-worker: 'Application Shell + data caching'
+perfomance-patterns: 'Asynchronous Partial Outputs'
+server-side-rendering: 'No'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Cloudfront'
+authors:
+    - name: 'Ivan'
+lighthouse: '91/100'
+interactive-em: '3.1s'
```

Comment:
EM first-interactive according to https://www.webpagetest.org/lighthouse.php?test=170607_FT_95b840f28ed8a72b031a8a06eb24ec68&run=3 (click through to the Lighthouse report on https://www.webpagetest.org/result/170607_FT_95b840f28ed8a72b031a8a06eb24ec68/) is 7.3s